### ISSUE TITLE
VG image features share

### ISSUE THREAD

**Issue by guoyang9** [peteanderson80/bottom-up-attention#53]
Could you please share the pre-trained image features from Visual Genome? This would greatly ease the affliction of ones without too much knowledge of caffe. Thannks.

**Comment by peteanderson80** [comment]
See https://github.com/peteanderson80/bottom-up-attention#pretrained-features

**Comment by guoyang9** [comment]
I mean the image features from Visual Genome. Though the pre-trained model is trained on this dataset, the image features can still be very useful in some specific tasks.

**Comment by duranze** [comment]
Pre-trained features do not contain semantic class labels of detections.  Hopefully the owner could upload the semantic info corresponding to 36 fixed(or not fixed) detected object. (Caffe is not compatible to my environment) @peteanderson80

**Comment by johnnyjana730** [comment]
I use docker to build up the env instead. I really try hard to do that. maybe you can try to nvidia docker. or you could find the way I could share my docker env to you.

**Comment by HYPJUDY** [comment]
Hi @johnnyjana730 , I am struggling with building up the env too. Could you kindly share me your docker env? Since I cannot find your email, could you contact me? (You can find my email in my GitHub page) Thanks!

**Comment by naykun** [comment]
I have the same need to get VG features, have you guys solved this problem?

**Comment by LMdeLiangMi** [comment]
same problem here...

**Comment by DesaleF** [comment]
@johnnyjana730 can you share me the your docker image please?
